USAGE.md

```diff
@@ -275,3 +276,16 @@ Above is an example of a constant violation entry in `deprecated_references.yml`
 * `components/merchant/app/public/merchant/generate_order.rb` - path to the file containing the violated constant
 
 Violations exist within the package that makes a violating reference. This means privacy violations of your package can be found listed in `deprecated_references.yml` files in the packages with the reference to a private constant.
+
+# Loading Extensions
```

Comment:
Later, when we support offenses formatter, checkers, and parsers to be configured using a consistent API, we can add more information about how to configure these within this section.